earthmover/__main__.py

```diff
@@ -162,15 +179,16 @@ def main(argv=None):
             force=args.force,
             skip_hashing=args.skip_hashing,
             cli_state_configs=cli_state_configs,
-            results_file=args.results_file
+            results_file=args.results_file,
+            skip_mkdir=args.command in ['clean', 'deps']
```

Comment:
Doesn't this command check misalign with the comparators on lines 191 and 217 (which use the values of the dictionary and not the keys)?

Reply:
Ah yep, you're totally right

Reply:
Fortunately I'm going to remove `skip_mkdir` anyway!